src/utils/colonyActions.ts

```diff
@@ -364,6 +368,14 @@ export const getSpecificActionValuesCheck = (
         currentColonyTokens?.slice(0).sort() || [],
       );
 
+      const addedSafe =
+        currentColonySafes.length > prevColonySafes.length
+          ? currentColonySafes.filter(
+              (safe) =>
+                !prevColonySafes.some((prevSafe) => isEqual(prevSafe, safe)),
```

Comment:
I think you can just do a simple check of `prevSafe.address === safe.address`.
I think that you can also use `.find` instead of filter, this would give you the element without the need to select `[0]` like you do now. But that's a nitpick.